ads/mobusi.md

```diff
@@ -0,0 +1,34 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Mobusi
+
+## Example
+
+```html
+  <amp-ad width="300" height="450"
+    type="mobusi"
+    src="https://play.leadzu.com/banner_request.php?m=1GBTSITE40382X2&a=&idtm=300x450&lgid=324132413241234123">
+  <div placeholder></div>
+  <div fallback>No ads</div>
+```
+
+## Configuration
+For semantics of configuration, please see ad network documentation.
+
+### Direct URL Call:
+* `src` - It is a custom URL for each area of the web of our publishers.
```

Comment:
Currently yes, but it is something that we are going to change soon, allowing other amp-ad formats